src/components/shared/TabNav/TabNav.tsx

```diff
@@ -11,13 +11,27 @@ interface Props {
 }
 
 interface Config {
+  showBadge?: boolean | null;
   title: string;
   to: string;
 }
 
 
 const TabNav = (props: Props): JSX.Element => {
-  const renderTabNavItems = props.config.map(({ title, to }: Config) => {
+  const renderTabNavItems = props.config.map(({ showBadge, title, to }: Config) => {
+    if (showBadge) {
+      return (
+        <div className="verification-needed-container">
+          <span className="verification-badge">!</span>
```

Comment:
Would we be able to get rid of the `showBadge` block and instead move the `verification-badge` span into the same place below? It would deduplicate some code and clarify what the condition really applies to

    {showBadge && <span className="verification-badge">!</span>}

Reply:
yea that would be ideal. I can conditionally render the span, but I'm having trouble finding ways to conditionally render parent div:

```
<div className="verification-needed-container">
     <span className="verification-badge">!</span>
     {children}
</div>
````
Maybe ternary might be a tad clearer?

Reply:
Does the parent `div` need conditional rendering? It seems like it could be a neutral container when the `span` isn't there, but I haven't really looked at CSS

If CSS does matter, you could try this on the parent:

    <div className={showBadge ? 'verification-needed-container' : ''}>

Reply:
good suggestion; I'll try to make the container more neutral and will comment when implemented!

Reply:
updated